fabric-tag-extensions-v0/src/main/java/net/fabricmc/fabric/mixin/tag/extension/AccessorDynamicRegistryManager.java

```diff
@@ -16,17 +16,19 @@
 
 package net.fabricmc.fabric.mixin.tag.extension;
 
+import java.util.Map;
+
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.gen.Accessor;
 
-import net.minecraft.fluid.Fluid;
-import net.minecraft.tag.FluidTags;
-import net.minecraft.tag.RequiredTagList;
+import net.minecraft.util.registry.DynamicRegistryManager;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
 
-@Mixin(FluidTags.class)
-public interface AccessorFluidTags {
-	@Accessor("REQUIRED_TAGS")
-	static RequiredTagList<Fluid> getRequiredTags() {
-		throw new UnsupportedOperationException();
+@Mixin(DynamicRegistryManager.class)
+public interface AccessorDynamicRegistryManager {
```

Comment:
Better named `DynamicRegistryManagerAccess`

Reply:
I thought we named accessor `<base class>Accessor` instead of Access? When did this change?